### DISCUSSION TITLE
Change Requests saying a page has changed when nothing has.

### DISCUSSION THREAD

**Discussion by Katerlad** [GitbookIO/community#381] [0. General bugs and questions]
Anytime I select a page from the outline, and then select a word on the page so that my typing cursor appears. Gitbook is marking that as a change. Even though nothing has changed... is this a bug?

This is very annoying, anytime we try to diff a change request it shows a ton of pages marked in blue, that was never changed. Just clicked on.

Edited: Added Steps to reproduce, Chrome Version, and video of replication!

I believe you have to make a change in a paragraph block on a page, and then while you cursor was in that block start clicking around to other pages, all pages that you click into after that will be marked as changed.

Steps to reproduce:

Step 1: Click Change Requests while viewing a Space.
Step 2: Far right corner click "new" button to create a new change request.
Step 3: Name your Change Request
Step 4: Click on a page you want to edit.
Step 5: Click on a paragraph block, and make a change aka add a few words.
Step 6: Click on a different page you want to view.
Step 7: Click inside that page so your cursor appears
Step 8: Repeat steps 6 and 7 on multiple pages.
Step 9: Click Diff View, to see that every page you clicked during steps 6 and 7 report a change with no changes showing on the page.

Here is My chrome Version:
![image](https://github.com/GitbookIO/community/assets/54480322/3e08edfc-c229-45f0-b5d5-ebca24b23a78)

Here is the video of the problem!
[Gitbook Bug.webm](https://github.com/GitbookIO/community/assets/54480322/1382dcac-8588-4633-94bc-27bda11f509c)

**Comment by aikoii** [comment]
Hi @Katerlad, thanks for your report! I tried reproducing the issue you described but didn't encounter it (on Chrome). Could you try [hard-refreshing](https://docs.gitbook.com/help/hard-refresh) your browser to get the latest version of the app and see if it resolves this problem?

**Reply by Katerlad** [reply]
Hello Aikoii, Thank you for responding!

I have done what you requested I am using Chrome, I did a hard refresh (ctrl + f5) and I made sure my browser was up to date. 
Unfortunatly the bug still exists. I did however figure out exact replication steps, and I know seeing is believing so I recorded a video.

I believe you have to make a change in a paragraph block on a page, and then while you cursor was in that block start clicking around to other pages, all pages that you click into after that will be marked as changed.

Steps to reproduce:

Step 1: Click Change Requests while viewing a Space.
Step 2: Far right corner click "new" button to create a new change request.
Step 3: Name your Change Request
Step 4: Click on a page you want to edit.
Step 5: Click on a paragraph block, and make a change aka add a few words.
Step 6: Click on a different page you want to view.
Step 7: Click inside that page so your cursor appears
Step 8: Repeat steps 6 and 7 on multiple pages.
Step 9: Click Diff View, to see that every page you clicked during steps 6 and 7 report a change with no changes showing on the page.

Here is My chrome Version:
![image](https://github.com/GitbookIO/community/assets/54480322/3e08edfc-c229-45f0-b5d5-ebca24b23a78)

Here is the video of the problem!
[Gitbook Bug.webm](https://github.com/GitbookIO/community/assets/54480322/1382dcac-8588-4633-94bc-27bda11f509c)

**Reply by aikoii** [reply]
Thanks so much for writing this up! We are taking a closer look 🙏
